### 🤖 AI Review Feedback
### 🤖 Automated Analysis

**Lint Report:**
File: /tmp/repo-wQxdxx/src/ai/ai.module.ts


File: /tmp/repo-wQxdxx/src/ai/ai.service.spec.ts


File: /tmp/repo-wQxdxx/src/ai/ai.service.ts


File: /tmp/repo-wQxdxx/src/app.controller.spec.ts


File: /tmp/repo-wQxdxx/src/app.controller.ts


File: /tmp/repo-wQxdxx/src/app.module.ts


File: /tmp/repo-wQxdxx/src/app.service.ts


File: /tmp/repo-wQxdxx/src/github/github.module.ts


File: /tmp/repo-wQxdxx/src/github/github.service.spec.ts


File: /tmp/repo-wQxdxx/src/github/github.service.ts


File: /tmp/repo-wQxdxx/src/main.ts
  [WARN] @typescript-eslint/no-floating-promises - Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator. (line 8)

File: /tmp/repo-wQxdxx/src/queue/ai.worker.ts


File: /tmp/repo-wQxdxx/src/queue/eslint.worker.ts


File: /tmp/repo-wQxdxx/src/queue/queue.module.ts


File: /tmp/repo-wQxdxx/src/queue/queue.service.spec.ts


File: /tmp/repo-wQxdxx/src/queue/queue.service.ts


File: /tmp/repo-wQxdxx/src/webhook/webhook.controller.spec.ts


File: /tmp/repo-wQxdxx/src/webhook/webhook.controller.ts
  [ERROR] @typescript-eslint/no-unsafe-assignment - Unsafe assignment of an `any` value. (line 12)
  [WARN] @typescript-eslint/no-unsafe-argument - Unsafe argument of type `any` assigned to a parameter of type `string`. (line 16)
  [ERROR] @typescript-eslint/no-unsafe-member-access - Unsafe member access .action on an `any` value. (line 16)
  [WARN] @typescript-eslint/no-unsafe-argument - Unsafe argument of type `any` assigned to a parameter of type `PullRequestPayload`. (line 18)

File: /tmp/repo-wQxdxx/src/webhook/webhook.module.ts


File: /tmp/repo-wQxdxx/src/webhook/webhook.service.spec.ts


File: /tmp/repo-wQxdxx/src/webhook/webhook.service.ts


File: /tmp/repo-wQxdxx/test/app.e2e-spec.ts


**Security Report:**
Package: @angular-devkit/core
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @angular-devkit/schematics
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @angular-devkit/schematics-cli
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @eslint/config-array
  Severity: high
  Title: undefined
  URL: undefined

Package: @eslint/eslintrc
  Severity: high
  Title: undefined
  URL: undefined

Package: @jest/core
  Severity: high
  Title: undefined
  URL: undefined

Package: @jest/expect
  Severity: high
  Title: undefined
  URL: undefined

Package: @jest/globals
  Severity: high
  Title: undefined
  URL: undefined

Package: @jest/reporters
  Severity: high
  Title: undefined
  URL: undefined

Package: @jest/transform
  Severity: high
  Title: undefined
  URL: undefined

Package: @nestjs/cli
  Severity: high
  Title: undefined
  URL: undefined

Package: @nestjs/schematics
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @octokit/core
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @octokit/graphql
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @octokit/plugin-paginate-rest
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @octokit/request
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @octokit/request-error
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @octokit/rest
  Severity: moderate
  Title: undefined
  URL: undefined

Package: @swc/cli
  Severity: high
  Title: undefined
  URL: undefined

Package: @typescript-eslint/eslint-plugin
  Severity: high
  Title: undefined
  URL: undefined

Package: @typescript-eslint/parser
  Severity: high
  Title: undefined
  URL: undefined

Package: @typescript-eslint/type-utils
  Severity: high
  Title: undefined
  URL: undefined

Package: @typescript-eslint/typescript-estree
  Severity: high
  Title: undefined
  URL: undefined

Package: @typescript-eslint/utils
  Severity: high
  Title: undefined
  URL: undefined

Package: ajv
  Severity: moderate
  Title: undefined
  URL: undefined

Package: babel-jest
  Severity: high
  Title: undefined
  URL: undefined

Package: babel-plugin-istanbul
  Severity: high
  Title: undefined
  URL: undefined

Package: create-jest
  Severity: high
  Title: undefined
  URL: undefined

Package: eslint
  Severity: high
  Title: undefined
  URL: undefined

Package: fork-ts-checker-webpack-plugin
  Severity: high
  Title: undefined
  URL: undefined

Package: glob
  Severity: high
  Title: undefined
  URL: undefined

Package: jest
  Severity: high
  Title: undefined
  URL: undefined

Package: jest-circus
  Severity: high
  Title: undefined
  URL: undefined

Package: jest-cli
  Severity: high
  Title: undefined
  URL: undefined

Package: jest-config
  Severity: high
  Title: undefined
  URL: undefined

Package: jest-resolve-dependencies
  Severity: high
  Title: undefined
  URL: undefined

Package: jest-runner
  Severity: high
  Title: undefined
  URL: undefined

Package: jest-runtime
  Severity: high
  Title: undefined
  URL: undefined

Package: jest-snapshot
  Severity: high
  Title: undefined
  URL: undefined

Package: minimatch
  Severity: high
  Title: undefined
  URL: undefined

Package: test-exclude
  Severity: high
  Title: undefined
  URL: undefined

Package: ts-jest
  Severity: high
  Title: undefined
  URL: undefined

Package: typescript-eslint
  Severity: high
  Title: undefined
  URL: undefined

--- a/src/ai/ai.service.ts
+++ b/src/ai/ai.service.ts
@@ -16,7 +16,7 @@ export class AiService {
   async analyzeDiff(diff: string): Promise<string> {
     try {
       const model = this.genAI.getGenerativeModel({
-        model: 'gemini-2.0-flash',
+        model: 'gemini-1.5-flash',
       });
 
       const prompt = `You are an expert code reviewer with deep expertise in software engineering best practices. Your role is to provide detailed, actionable feedback on code changes.

